extensions/amp-story/1.0/amp-story.js

```diff
@@ -580,6 +581,12 @@ export class AmpStory extends AMP.BaseElement {
 
     this.initializeBookend_();
 
+    const {document} = this.win;
+
+    if (isRTL(document)) {
+      this.storeService_.dispatch(Action.TOGGLE_RTL, true);
```

Comment:
I think we should try to set this state as soon as possible in the code execution, like we do for `desktop`.

Right now, the RTL state is set after some components are initialized, which means you'd have to watch for `RTL_STATE` updates from every component, to later set the `dir="rtl"` attributes.
This can be avoided if you can find a way to instantiate the state right away with the proper value, or set it first thing in the `buildCallback` of `amp-story`.

Reply:
Good idea